### PR TITLE
Style fixes to Logo Guidance page

### PR DESCRIPTION
**Pull request by sshami**
Hi @cmc333333 - when you get the chance, could you merge and deploy to Staging? Thanks!

### CODE REVIEW

File: src/css/modules/_panel.scss [hunk 1]
```diff
@@ -209,55 +209,29 @@
...
+  &.simplepage {
+    ul {
+      color: $black;
+      max-width: 45em;
+      margin:0 auto;
+      font-size:1.2em;
+      padding-left:3.5em;
+      padding-right:2.5em;
+      span {
+        font-family: Archer, Archer SSm A, Archer SSm B, serif;
```
Comment (cmc333333) [inline src/css/modules/_panel.scss:222]: Probably a good idea to use a variable if re-using this font string often

**Comment by cmc333333** [comment]
@sshami Pushed to staging. Let me know when you want it to go to prod

**Comment by sshami** [comment]
Thanks @cmc333333 Good for production whenever you get the chance

**Comment by cmc333333** [comment]
@sshami pushed

**Pull request merged**